specifications/api/swagger.yaml

```diff
@@ -2085,6 +2085,11 @@ components:
             description: Only for 'Byron' style.
             type: string
             format: base16
+          address_type:
+            description: The raw type field of the address.
```

Comment:
I wonder if we should not somewhere here also add the table as in cardano-addresses describing codes?

Reply:
Added table and some tests.